server/routes/bookedtoday/unexpectedArrivals/validation/searchForExistingRecordsValidation.ts

```diff
@@ -22,6 +22,10 @@ const SearchForExistingRecordsValidator: Validator = ({
     ]
   }
 
+  if ((lastName && prisonNumber) || (lastName && pncNumber)) {
+    return []
+  }
+
```

Comment:
It seems logical that if a user can search using either last name plus dob, prison number or pnc number, then a search using last name and either prison or pnc number should be valid.